Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracks_cff.py

```diff
@@ -14,9 +14,9 @@
 from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemTimingRawToDigi as _totemTimingRawToDigi
 from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import ctppsPixelDigis as _ctppsPixelDigis
 
-ctppsDiamondRawToDigiAlCaRecoProducer  = _ctppsDiamondRawToDigi.clone(rawDataTag = 'hltPPSCalibrationRaw')
-totemTimingRawToDigiAlCaRecoProducer = _totemTimingRawToDigi.clone(rawDataTag = 'hltPPSCalibrationRaw')
-ctppsPixelDigisAlCaRecoProducer = _ctppsPixelDigis.clone(inputLabel = 'hltPPSCalibrationRaw')
+ctppsDiamondRawToDigiAlCaRecoProducer  = _ctppsDiamondRawToDigi.clone(rawDataTag = 'rawDataCollector')
+totemTimingRawToDigiAlCaRecoProducer = _totemTimingRawToDigi.clone(rawDataTag = 'rawDataCollector')
+ctppsPixelDigisAlCaRecoProducer = _ctppsPixelDigis.clone(inputLabel = 'rawDataCollector')
```

Comment:
Why where these changed from `hltPPSCalibrationRaw` to `rawDataCollector`?
This is now causing the failures observed in the Relvals-input for `1043.0,1044.0,1045.0`
And it's indeed spotted also by the (failing) PPS unitTests:
   - Calibration/PPSAlCaRecoProducer/test/testExpressPPSAlCaRecoProducer
   - Calibration/PPSAlCaRecoProducer/test/testPromptPPSAlCaRecoProducer

Reply:
it was a change needed for a temporary test, it is fixed in https://github.com/cms-sw/cmssw/pull/42515/commits/0d1170c5b303f20b1b06568b994a27f55a1d017b